Manifests/index

```diff
@@ -1078,11 +1078,11 @@
 		<key>com.apple.SetupAssistant.managed</key>
 		<dict>
 			<key>modified</key>
-			<date>2019-06-26T01:07:33Z</date>
+			<date>2021-09-15T08:15:39Z</date>
```

Comment:
Let's exclude the index file from this PR, since we update this after we merge everything in.

Reply:
See cb779c7.